docs/plugins/premium/config-source/aws-secrets.md

```
@@ -0,0 +1,142 @@
+head:
+  - - meta
+    - name: description
+      content: A powerful plugin for Ts.ED to manage your application configuration with AWS Secrets manager and sync changes in real time.
+  - - meta
+    - name: keywords
+      content: Ts.ED plugins premium aws config-source secrets manager watch-mode hot-reload
+---
+
+# AWS Secrets Config Source
+
+<Banner src="/aws.png" height="200" href="https://github.com/luin/ioredis"></Banner>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: Banner

(MD033, no-inline-html)